src/untils/expenses.js

```diff
@@ -0,0 +1,24 @@
+const isExpenseValid = (expense) => {
+  const { title, amount, category, spentAt } = expense;
+
+  return !!(title && amount && category && spentAt);
```

Comment:
The `isExpenseValid` function only checks for the presence of fields but does not validate their types or values. Consider adding checks to ensure `amount` is a positive number, `spentAt` is a valid date, and `title` and `category` are non-empty strings.